code/modules/projectiles/projectile/magic.dm

```diff
@@ -218,17 +222,19 @@
 							/mob/living/simple_animal/hostile/stickman/dog,
 							/mob/living/simple_animal/hostile/megafauna/dragon/lesser,
 							/mob/living/simple_animal/hostile/gorilla,
+							/mob/living/simple_animal/hostile/lightgeist/healing,
 							/mob/living/simple_animal/parrot,
 							/mob/living/simple_animal/pet/dog/corgi,
 							/mob/living/simple_animal/crab,
 							/mob/living/simple_animal/pet/dog/pug,
 							/mob/living/simple_animal/pet/cat,
-							/mob/living/simple_animal/mouse,
-							/mob/living/simple_animal/chicken,
+							/mob/living/simple_animal/pet/cat/space,
+							/mob/living/simple_animal/chocobo,
+							/mob/living/simple_animal/hostile/retaliate/goose/vomit,
```

Comment:
sorry, no birdboat, this one is buggy currently

Reply:
Removed birdboat